code/__byond_version_compat.dm

```diff
@@ -28,7 +28,7 @@
 // and so this check is in place to stop serious damage.
 // That being said, if you really are ready, you can give YES_I_WANT_515 to TGS.
 #if !defined(YES_I_WANT_515) && DM_VERSION >= 515
-#error We do not yet completely support BYOND 515.
+#warn We do not yet completely support BYOND 515.
```

Comment:
This needs to be reverted

Reply:
This was when this error broke StrongDMM

Reply:
This is when this error broke StrongDMM compilation

975bc70